code/modules/mob/living/carbon/alien/adult/alien_powers.dm

```diff
@@ -200,6 +200,8 @@ Doesn't work on other aliens/AI.*/
 	desc = "Drench an object in acid, destroying it over time."
 	button_icon_state = "alien_acid"
 	plasma_cost = 200
+	var/corrosion_acid_power = 200
+	var/corrosion_acid_volume = 1000
```

Comment:
You have to autodoc new class variables https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md#documenting-a-variabledefine

Reply:
thanks